website/pages/consul-on-kubernetes/index.tsx

```diff
@@ -9,15 +9,28 @@ import DocsList from 'components/docs-list'
 import s from './style.module.css'
 
 export default function ConsulOnKubernetesPage() {
+  const pageDescription =
+    'Consul is a robust service mesh for discovering and securely connecting applications on Kubernetes.'
+  const pageTitle = 'Consul on Kubernetes'
+
   return (
     <div>
-      <Head>
-        <title key="title">Consul on Kubernetes</title>
+      <ReactHead
+        title={pageTitle}
+        pageName={pageTitle}
+        description={pageDescription}
+        image="https://www.consul.io/img/consul-on-kubernetes-share-image.png"
```

Comment:
```suggestion
        image="/img/consul-on-kubernetes-share-image.png"
```

Should we roll with a relative URL here (would allow us to test in preview, I think)

Reply:
(oh and same with the og:image below)